packages/sdk/src/reliable_channel/reliable_channel.ts

```diff
@@ -512,18 +551,60 @@ export class ReliableChannel<
     if (this.sweepInBufInterval) clearInterval(this.sweepInBufInterval);
   }
 
+  private startRepairSweepLoop(): void {
+    this.stopRepairSweepLoop();
+    this.sweepRepairInterval = setInterval(() => {
+      void this.messageChannel
+        .sweepRepairIncomingBuffer(async (message) => {
+          // Rebroadcast the repair message
+          const wakuMessage = { payload: message.encode() };
+          const result = await this._send(this.encoder, wakuMessage);
+          return result.failures.length === 0;
+        })
+        .catch((err) => {
+          log.error("error encountered when sweeping repair buffer", err);
+        });
+    }, DEFAULT_SWEEP_REPAIR_INTERVAL_MS);
+  }
+
+  private stopRepairSweepLoop(): void {
+    if (this.sweepRepairInterval) clearInterval(this.sweepRepairInterval);
```

Comment:
```suggestion
    if (this.sweepRepairInterval) {
         clearInterval(this.sweepRepairInterval);
         this.sweepInBufInterval = undefined;
   }
```

It's apparently better.